modules/tinymce/src/core/main/ts/api/Tinymce.ts

```diff
@@ -43,7 +43,6 @@ import Resource from './Resource';
 import Shortcuts, { ShortcutsConstructor } from './Shortcuts';
 import ThemeManager from './ThemeManager';
 import UndoManager from './UndoManager';
-import Class from './util/Class';
```

Comment:
This might have come up before in another PR but should we remove the reference in `globals.json` file as well? Looking at that file, some of the other removed APIs are still in the list so maybe that can be done separately.